### DISCUSSION TITLE
Platform Specific XAML

### DISCUSSION THREAD

**Discussion by lylerolleman** [AvaloniaUI/Avalonia#12632] [General]
Quick question, is there an Avalonia equivalent to Xamarin's OnPlatform?

I'd like to be able to have platform specific markup/values/styles but I can't seem to find anything, other than the "heavy handed" solution of alternate views or or binding the properties to a VM property with platform/OS specific values

**Comment by stevemonaco** [comment]
Yes, it's also `OnPlatform` (Avalonia 11+). Check out the Control Catalog for a [sample](https://github.com/AvaloniaUI/Avalonia/blob/master/samples/ControlCatalog/Pages/PlatformInfoPage.xaml). There aren't any docs yet for this (AFAIK), but it's being tracked over at https://github.com/AvaloniaUI/avalonia-docs/issues/20
